packages/api/src/fetchSvgs.ts

```diff
@@ -0,0 +1,39 @@
+import axios from 'axios'
+
+const DEFAULT_SVG = '<svg></svg>'
+
+export async function fetchSvgs (
+  networksToFech: Array<string>
+): Promise<{ [key: string]: string }> {
+  const BASE_URL =
+    'https://raw.githubusercontent.com/witnet/data-feeds-explorer/main/packages/ui/assets/svg/'
+
+  const logosUrls = networksToFech.map((networkToFech: string) => {
+    return `${BASE_URL}${networkToFech}.svg`
+  })
+
+  // Fech all logos from github
+  const promises = logosUrls.map(url => axios.get(url))
+  return new Promise(resolve => {
+    Promise.allSettled(promises).then(results => {
+      const svgs = results.map((result, index) => {
+        if (result.status === 'fulfilled') {
```

Comment:
The status of fetching the SVG sometimes is rejected and it returns an empty state.